informaten.com.hosting.json

```diff
@@ -0,0 +1,85 @@
+{
+  "providerId": "informaten.com",
+  "providerName": "Informaten",
+  "serviceId": "hosting",
+  "serviceName": "Webhosting on Informaten.com",
+  "version": 1,
+  "syncPubKeyDomain": "informaten.com",
+  "logoUrl": "https://informaten.com/assets/logo_dark.png",
+  "description": "Enables a domain to work with Informaten webhosting",
+  "variableDescription": "ip and ip6 are the v4/v6 address of the webspace server, domain is the domain this template is applied to and hostname is the hostname of the webspace server",
+  "records": [
+    {
+      "type": "A",
+      "host": "@",
+      "pointsTo": "%ip%",
+      "essential": "OnApply",
+      "ttl": 3600
+    },
+    {
+      "type": "AAAA",
+      "host": "@",
+      "pointsTo": "%ip6%",
+      "essential": "OnApply",
+      "ttl": 3600
+    },
+    {
+      "type": "CNAME",
+      "host": "www",
+      "pointsTo": "@",
+      "essential": "OnApply",
+      "ttl": 3600
+    },
+    {
+      "type": "MX",
+      "host": "@",
+      "pointsTo": "mail",
```

Comment:
My previous advice was incomplete.
It should have been `mail.%fqdn%.`

BTW: Template testing is possible here:
https://pdnsadmin.revproxy.short-lived.de/dc/free/templateedit

Reply:
no problem, i fixed it